include/julia_wrapper.hpp

```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#ifdef _WIN32
+/* julia.h includes Windows headers that are not self contained, so this
+ * wrapper header includes Windows.h before includeing julia.h, so everything
+ * is setup for the other headers to actually work */
```

Comment:
```suggestion
// julia.h includes Windows headers that are not self contained, this
// wrapper header includes Windows.h before includeing julia.h, so everything
// is setup for the other headers to actually work
```
(my style guide prohibits multi-line comments in release-version code)